README.md

```diff
@@ -227,6 +227,12 @@ Copyright:
  © 2013-2018 Alexey Khokholov (Nuke.YKT).  
 License: [GPL-2.0+](https://www.gnu.org/licenses/old-licenses/gpl-2.0.html)
 
+Files: `third-party/pffft/*`  
```

Comment:
Rumble still needs mention in our feature list in README.md.

Reply:
Fixed here: https://github.com/fabiangreffrath/woof/pull/1876/commits/7faa0c130b203b7b28e66afad32d8d6a2d58d296